docs/mdbook/src/android/map-options-guide.md

```diff
@@ -0,0 +1,308 @@
+# MapLibreMapOptions
```

Comment:
Give the article a descriptive name that explains the contents of the article. "Ways to Configure the Map" or something similar.